test/eway_rapid/integration/transaction/query_transaction_test.rb

```diff
@@ -33,7 +33,11 @@ def test_valid_input
 
   def test_blank_input
     res = @client.query_transaction_by_access_code('')
-    assert_nil(res.transaction)
+    if IntegrationTest.get_version >= 47
+      assert_nil(res.transaction)
+    else
+      assert_nil(res.transaction_status.transaction_id)
+    end
```

Comment:
Tests use the sandbox API.
In case empty id is provided, it chages it to the `/Transaction/0`
```ruby
@client.query_transaction_by_access_code('')
```

The 47 version returns a different response than 31 and 40 version
![image](https://github.com/warwick-schaffer/eway-rapid-ruby/assets/52406903/ba8b89e4-fb0f-4576-8ed4-b9d76fc5fed8)

So in other words the 47 version natively pass the tests, but the 31 and 40 version fail. However there are many data are returned in the case of 31 and 40 versiosn, the **transaction_id** colums is null there (looks like its present when the requests was created by the user)
I changed the test to check the **transaction_id** for those versions